### ISSUE TITLE
Review UNFCCC, EUCoM, and contextual data

### ISSUE THREAD

**Issue by lgloege** [Open-Earth-Foundation/OpenClimate-harmonize#12]
@evanp , please review the csv files in:
-  `/data_emissions/country/UNFCCC`
- `/data_emissions/city/EUCoM`
- `/data_contextual/country/GDP`
- `/data_contextual/country/population`
- `/data_contextual/country/territory`

Thanks! :)

**Comment by evanp** [comment]
I added DataSourceTag records for UNFCCC; see the PR!

**Comment by evanp** [comment]
It looks like there are two data sources mixed in the eucom directory? I have been doing one data source per directory. If one of them changes, it seems like you'd only want to update once.

**Comment by evanp** [comment]
We already included the GDP and Population. I think for Territory, you should drop the boundary polygon which we don't use right now, and just include the area, which we do.

**Comment by lgloege** [comment]
Yes, I drop the polygon from Territory.

**Comment by lgloege** [comment]
Yes, Zhi Yi created the EUCoM dataset by compiling three different datasources. Since I knew which datasource each record came from I chose to retain that information. 

There are two publishers and three datasources.
